src/ProjectTemplates/test/IdentityUIPackageTest.cs

```diff
@@ -118,12 +65,12 @@ public IdentityUIPackageTest(ProjectFactoryFixture projectFactory, ITestOutputHe
             "Identity/lib/jquery-validation-unobtrusive/LICENSE.txt",
         };
 
-        [ConditionalTheory]
-        [MemberData(nameof(MSBuildIdentityUIPackageOptions))]
+        [ConditionalFact]
         [SkipOnHelix("cert failure", Queues = "OSX.1014.Amd64;OSX.1014.Amd64.Open")]
         [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/19716")]
-        public async Task IdentityUIPackage_WorksWithDifferentOptions(IDictionary<string, string> packageOptions, string versionValidator, string[] expectedFiles)
+        public async Task IdentityUIPackage_WorksWithDifferentOptions()
```

Comment:
Should this be unquarantined now? The linked issue is closed.

Reply:
The procedure as I understand it is that the Build-Buddy will unquarantine things that haven't failed in the Quarantine test in 30 days. Given that it was an (educated) quess that the test was fixed I think it prudent to follow that process rather than #YOLO'ing it back in.

Reply:
Oh I thought that was only for flaky tests. Not for surely failing tests due to path issues. But I could be wrong. @BrennanConroy?

Reply:
> The procedure as I understand it is that the Build-Buddy will unquarantine things that haven't failed in the Quarantine test in 30 days. Given that it was an (educated) guess that the test was fixed I think it prudent to follow that process rather than #YOLO'ing it back in.

Also, shouldn't we keep the linked issue open to track unquarantining it?

Reply:
Better to be safe and leave quarantined. If the test behaves nicely it'll be set free in ~30 days 😄

Reply:
The test was originally Quarantined for a Flaky issue, which is hypothetically fixed. The "for-sure-fail" happened after (and was unnoticed due to  quarantine).

If I recall I suggested the same thing and was told that the dashboard reporting it as 100% success over 30 days is the tracking for it.

Reply:
Thanks for the clarifications.